### ISSUE TITLE
How can I modify css?

### ISSUE THREAD

**Issue by ramihan** [mstaal/msal_streamlit_authentication#25]
This library runs in iframe, user css configuration in streamlit is not inherited when I changed a class or id property.
How can I change the log in / log out button's style?

**Comment by mstaal** [comment]
Yeah, you are right. I think I will try to look into options of letting it
inherit styles from Streamlit.

I am in the midst of upgrading the underlying MSAL library. I hope to have
it ready soon. And then I will put my focus onto the styling. Sorry for the
inconvenience.

søn. 5. nov. 2023 kl. 10.38 skrev Rami(라미) ***@***.***>:

**Comment by andreped** [comment]
> I hope to have it ready soon.

This would be of interest for me as well. I can always try to do CSS styling myself to resolve this, but customing themeing through CSS in Streamlit is quite tricky, especially if you want the exact same theme styling. if we could just inherit the style, this would be a lot easier :]
